lib/Util.ts

```diff
@@ -161,7 +167,7 @@ export class Util {
           writeStream.on('close', resolve);
           writeStream.on('error', reject);
         });*/
-        writeFileSync(cachePathLocal + '.url', response.url);
```

Comment:
We need the final URL though, since this URL might have changed with redirects.

Reply:
Fixed.